corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js

```diff
@@ -429,8 +429,10 @@ hqDefine("cloudcare/js/formplayer/menus/views", function () {
             //do something
         },
 
-        continueAction: function (e) {
-            // TODO: implement function to send selectedCaseIds
+        continueAction: function () {
+            if (selectedCaseIds.length > 0) { // TODO: make button disabled if no cases selected
```

Comment:
Just noting this - is this for this PR or another one?

Reply:
I was imagining it would be part of this PR but wanted to prioritize the non-styling aspects first. thoughts?

Reply:
sounds fine to me